framework/JobHandler.py

```diff
@@ -211,7 +211,7 @@ def __initializeRay(self):
             self.raiseADebug("Head host IP      :", address)
             self.raiseADebug("Head redis pass   :", redisPassword)
           ## Get servers and run ray remote listener
-          servers = self.runInfoDict['remoteNodes'] if self.rayInstanciatedOutside else self.__runRemoteListeningSockets(address, localHostName, redisPassword, db)
+          servers = self.runInfoDict['remoteNodes'] if self.rayInstanciatedOutside else self.__runRemoteListeningSockets(address, localHostName, redisPassword)
```

Comment:
is this modification part of this PR?

Reply:
This is an error in our code, the 'db' is not a valid param for  `self.__runRemoteListeningSockets`